.drone.yml

```diff
@@ -331,6 +331,27 @@ services:
       - cd /tmp/direct-start-worker
       - pip3 install -r src/requirements.txt
       - python3 src/worker.py
+  - name: opencti-test-worker
+    image: nikolaik/python-nodejs:python3.11-nodejs20-alpine
+    volumes:
+      - name: cache-node-backend
+        path: /tmp/direct-start-platform/node_modules
+    environment:
+      OPENCTI_URL: http://api-tests:4010
+      OPENCTI_TOKEN: bfa014e0-e02e-4aa6-a42b-603b19dcf159
+      WORKER_LOG_LEVEL: info
+    commands:
+      - sleep 10
+      - cp -a opencti-worker /tmp/opencti-test-worker
+      - apk add build-base git libffi-dev cargo
+      - cd "$DRONE_WORKSPACE/client-python"
+      - echo "[INFO] using client-python on branch $(git branch --show-current)"
+      - pip install -r requirements.txt
+      - pip install -e .[dev,doc]
+      - while ! nc -z api-tests 4010 ; do sleep 1 ; done
+      - cd /tmp/opencti-test-worker
+      - pip install -r src/requirements.txt
```

Comment:
I think that pip install can be run before waiting on backend with `while ! nc -z api-tests 4010 ; do sleep 1 ; done`, it will save some few time.